README.md

```diff
@@ -35,41 +35,39 @@ You are good to go! Press the extension icon on any Chess.com game to have it im
 ![chesscom-to-lichess-example](https://user-images.githubusercontent.com/34536619/120174245-baade080-c1b9-11eb-8aa8-f21bd3d3bc89.gif)
 
 
-## **FAQ**
+## FAQ
+**Q: Can you run this on Mozilla Firefox?**  
+A: Yes! You can actually install it on Firefox and it should work as inteded.
 
-Q: Can you run this on Mozilla FireFox?  
-A: Yes! You can actually install it on FireFox and it should work as inteded.
-
-Q: Can you run this on Microsoft Edge?  
+**Q: Can you run this on Microsoft Edge?**  
 A: Technically Microsoft Edge supports chrome extensions, but I have not tested it.
 
-Q: Does this help someone cheat?  
+**Q: Does this help someone cheat?**  
 A: No, you can only import and analyse finished games.
 
+**Q: Why is it not on the chrome web store? Are you trying to hide something?**  
+A: To put an extension up on a chrome web store, I would need to make a developer account there and pay $5.
+As this was a weekend project, I didn't want to spend $5 on it. However, I will put it up there soon-ish.
 
-Q: Why is it not on the chrome web store? Are you trying to hide something?  
-A: To put an extension up on a chrome web store I would need to make a developer account there and pay $5. As this was a weekend project I didn't want to spend $5 on it. However I will put it up there soon-ish.
-
-
-Q: Why not just play on Lichess.org? What is the point of playing on chess.com if you will just import all the games?  
-A: I just prefer to play on chess.com. I do also play on Lichess.org but chess.com is my main account. Since I still mostly use the Lichess tools it is more convenient for me and people in a similar situation to just quickly import games to Lichess instead.
-
+**Q: Why not just play on Lichess.org? What is the point of playing on chess.com if you will just import all the games?**  
+A: I just prefer to play on chess.com. I do also play on Lichess.org but chess.com is my main account.
+Since I still mostly use the Lichess tools,
+it is more convenient for me and people in a similar situation to just quickly import games to Lichess instead.
 
-Q: Will this get me banned?  
+**Q: Will this get me banned?**  
 A: No, it will not.
 
+**Q: Isn't it unethical to use Lichess' computer analysis if you do not play there?**  
```

Comment:
Lichess recommends a similar extension [here](https://lichess.org/page/extend), so they don't seem to mind people using their analysis for chess.com games. Maybe we should mention that?

Reply:
That is a good idea, will look to include it